lib/electric_slide/call_queue.rb

```diff
@@ -388,8 +388,12 @@ def call_agent(agent, queued_call)
       queued_call.on_end { ignoring_ended_calls { agent_call.hangup } }
 
       agent_call.on_unjoined do
-       ignoring_ended_calls { agent_call.hangup }
-       ignoring_ended_calls { queued_call.hangup }
+        ignoring_ended_calls { agent_call.hangup }
+
+        ignoring_ended_calls do
+          queued_call.hangup if queued_call[:electric_slide_transferring_to].blank?
```

Comment:
Would you add an INFO level log here, indicating that the queued call is transferring and so it is being kept alive? From past experience, this can cause calls to "mysteriously" be abandoned. Logging that we are intentionally not terminating the call will help diagnose future problems.

Reply:
Addressed in f123cfdab7f48cf31ae2b63514c1f082a86a9e52.